src/sentry/seer/issue_summary.py

```diff
@@ -233,6 +233,44 @@ def _get_trace_connected_issues(event: GroupEvent) -> list[Group]:
     return connected_issues
 
 
+def _run_automation(
+    group: Group,
+    user: User | RpcUser | AnonymousUser,
+    event: GroupEvent,
+    source: str,
+):
+    if features.has(
+        "organizations:trigger-autofix-on-issue-summary", group.organization, actor=user
+    ):
+        with sentry_sdk.start_span(op="ai_summary.generate_fixability_score"):
+            try:
+                issue_summary = _generate_fixability_score(group.id)
+            except Exception:
+                logger.exception("Error generating fixability score", extra={"group_id": group.id})
```

Comment:
We should return early here. Otherwise, when we access `issue_summary.scores` later on we'll end up with an error, since the variable isn't defined.